lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,139 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        lines_of_code: integer?,
+        last_committed_at: string
+      )
+    end
+
+    def analyzer_version
+      Runners::VERSION
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      analyze_last_committed_at(changes.changed_paths)
+      analyze_lines_of_code(changes.changed_paths)
+
+      Results::Success.new(
+        guid: guid,
+        analyzer: analyzer,
+        issues: changes.changed_paths.map { |path| generate_issue(path) }
+      )
+    end
+
+    private
+
+    def generate_issue(path)
+      filename = path.to_path
+      loc = lines_of_code[filename]
+      commit = last_committed_at[filename]
```

Comment:
[suggestion] How about using `fetch` to avoid nullable `commit`?

```suggestion
      commit = last_committed_at.fetch(filename)
```

Reply:
fix: e2466ef97f58e767838084d349323953307597a0